src/handler.py

```diff
@@ -0,0 +1,54 @@
+from os import path
+from src.exportImage import ExportImage
+from src.helper import Helper
+from logger.jsonLogger import Logger
+from src.config import read_json
+from pyority_queue.task_handler import *
+
+
+class Handler:
+    def __init__(self, asyncio_event_loop):
+        self.log = Logger.get_logger_instance()
+        self.__helper = Helper()
+        self.loop = asyncio_event_loop
+        current_dir_path = path.dirname(__file__)
+        config_path = path.join(current_dir_path, '../config/production.json')
+        self.__config = read_json(config_path)
+        self.queue_handler = TaskHandler(self.__config["server"]["job_type"], self.__config["server"]["task_type"],
+                                         self.__config["server"]["job_manager_url"], self.__config["server"]["heartbeat_manager_url"],
+                                         self.__config["server"]["heartbeat_interval_ms"], self.log)
+        self.__exportImage = ExportImage(self.queue_handler, self.loop)
+
+    async def handle_tasks(self):
+        try:
+            while True:
+                task = await self.queue_handler.dequeue(5)
```

Comment:
5? no hard coded values - put in a static class member

Reply:
fixed in a2ede34